pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
   flutter:
     sdk: flutter
   highlightable: ^1.0.5
+  rxdart: ^0.27.7
```

Comment:
Why we need this?

Reply:
RxDart is a package for handling streams - that's what I used it for.

Reply:
Cannot be implemented with default dart streams?